recipes-kernel/linux/linux-toradex/0001-mwifiex-Set-WIPHY_FLAG_NETNS_OK.patch

```diff
@@ -0,0 +1,27 @@
+From ce9a141a6daa640eb2e05964fbc3afd80919656b Mon Sep 17 00:00:00 2001
+From: Johannes Wiesboeck <johannes.wiesboeck@aisec.fraunhofer.de>
+Date: Wed, 26 Apr 2023 11:13:03 +0200
+Subject: [PATCH] mwifiex: Set WIPHY_FLAG_NETNS_OK
```

Comment:
Use the patch which was submitted upstream.

Reply:
done